impexp-client/src/main/java/org/citydb/modules/kml/gui/preferences/GeneralPanel.java

```diff
@@ -429,7 +458,10 @@ private void setEnabledComponents() {
 		gltfConverterBrowseText.setEnabled(createGltfCheckbox.isSelected());
 		gltfConverterBrowseButton.setEnabled(createGltfCheckbox.isSelected());			
 		notCreateColladaCheckbox.setEnabled(createGltfCheckbox.isSelected());
-		embedTexturesInGltfCheckbox.setEnabled(createGltfCheckbox.isSelected());
+		// if glb is enabled, always embed textures
+		embedTexturesInGltfCheckbox.setEnabled(createGltfCheckbox.isSelected() && !exportGltfBinary.isSelected());
```

Comment:
Was NOT moved to branch [glb-draco-support](https://github.com/3dcitydb/importer-exporter/tree/glb-draco-support).